src/App.js

```diff
@@ -19,15 +29,26 @@ export class App extends Component {
               <div className="control">
                 <input
                   type="text"
+                  value={this.state.text}
                   id="search-query"
                   className="input"
                   placeholder="Type search word"
+                  onChange={this.handleChange}
                 />
               </div>
             </div>
           </div>
 
-          <MoviesList movies={moviesFromServer} />
+          <MoviesList
+            movies={
+              [...moviesFromServer].filter((item) => {
```

Comment:
```suggestion
              moviesFromServer.filter((item) => {
```
 filter не мутирующий метод